regions:
    ros__parameters:

        pointcloud_topic: "/ouster/points"

        front:
            point_thresh: 5.0
            x_min: -0.5
            x_max: 0.5
            y_min: 0.0
            y_max: 2.0
            z_min: 0.0
            z_max: 2.0
            color:
                    r: 255.0
                    g: 0.0
                    b: 0.0
                    a: 0.3

        frontright:
            point_thresh: 5.0
            x_min: -2.5
            x_max: -0.5
            y_min: 0.0
            y_max: 2.0
            z_min: 0.0
            z_max: 2.0
            color:
                    r: 0.0
                    g: 255.0
                    b: 0.0
                    a: 0.3

        frontleft:
            point_thresh: 5.0
            x_min: 0.5
            x_max: 2.5
            y_min: 0.0
            y_max: 2.0
            z_min: 0.0
            z_max: 2.0
            color:
                    r: 0.0
                    g: 0.0
                    b: 255.0
                    a: 0.3

        rear:
            point_thresh: 5.0
            x_min: -0.5
            x_max: 0.5
            y_min: -2.0
            y_max: 0.0
            z_min: 0.0
            z_max: 2.0
            color:
                    r: 0.0
                    g: 0.0
                    b: 0.0
                    a: 0.3